src/main.cpp

```diff
@@ -32,6 +32,10 @@ void hkOnTouchMove(void* _, SCallbackInfo& cbinfo, std::any e) {
     cbinfo.cancelled = g_pGestureManager->onTouchMove(ev);
 }
 
+void hkOnOppositeGesture(void* _, SCallbackInfo& cbinfo, std::any e) {
+    HyprlandAPI::addNotification(PHANDLE, "[hyprgrass] Is this vertical? ", CColor{1.0, 0.2, 0.2, 1.0}, 500);
```

Comment:
```suggestion
    HyprlandAPI::addNotification(PHANDLE, "[hyprgrass] Is this vertical? ", CColor{1.0, 0.2, 0.2, 1.0}, 500);
    cbinfo.cancelled = true;
```

Reply:
we'll see if there's a better way to this other than abusing cbinfo, but for now this is fine